### PR TITLE
Add escape menu

### LINKED ISSUE #100: Option to quit out of the running game except alt+f4/pkill

### PR DESCRIPTION
**Pull request by jakobbbb**
Fixes #100

### CODE REVIEW

File: Assets/Menu/PauseMenu/PauseMenu.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        Time.timeScale = 0.0f;
+        Debug.Log("Options");
+    }
+    public void OptionBack(){
+        optionsMenu.SetActive(false);
+        StartCoroutine(UIUtility.SelectButtonLater(button));
+    }
+    public void QuitGame(){
+        SceneManager.LoadScene("MenuScene");
+    }
```
Comment (jakobbbb) [inline Assets/Menu/PauseMenu/PauseMenu.cs:62]: @Mielein Reason for menu not appearing is that `SceneTransitionCanvas` is faded to black.
I think it'd be best to use a portal to handle the transition to the main menu -- that way we'll have a nice transition.

**Review by jakobbbb — COMMENTED** [review]
Note: We'll need to throw away any changes to the mall scenes, since they've been smolified on main.  But as long as everything's in the prefabs it should be fine :3

**Comment by jakobbbb** [comment]
TODO:
- [ ] All text has gone orange for some reason
- [ ] Dialogue keypresses should be ignored when pausemenu is open
- [ ] Portal keypresses should be ignore when pausemenu is open
- [ ] Inventory should be closed (and unable to be opened) when pausemenu is open

**Comment by Mielein** [comment]
> TODO:
> 
>     * [ ]  All text has gone orange for some reason
> 
>     * [ ]  Dialogue keypresses should be ignored when pausemenu is open
> 
>     * [ ]  Portal keypresses should be ignore when pausemenu is open
> 
>     * [ ]  Inventory should be closed (and unable to be opened) when pausemenu is open
> 
> 
> I've rebased, dropping all changes to `mall_cute`, to resolve any conflicts. @Mielein I can take over the keypress ignore stuff, maybe you can look into the orange text thing?

Yee sure thing! :3

**Pull request merged**